R/utils.R

```diff
@@ -371,3 +372,9 @@ get_git_root <- function() {
 
   git_root
 }
+
+gh_cat <- function(string) {
+  if (Sys.getenv("GITHUB_ACTIONS") == TRUE) {
```

Comment:
You don't need the equality comparison if the condition is `TRUE`. Also, I think these are always strings 

```r
Sys.setenv("GITHUB_ACTIONS" = TRUE)
Sys.getenv("GITHUB_ACTIONS")
#> [1] "TRUE"
```
So I suggest you do `as.logical(upper(Sys.getenv("GITHUB_ACTIONS")))`.

Reply:
Yeah being explicit might be better than relying on implicit casting...